econml/validate/utils.py

```diff
@@ -98,14 +101,25 @@ def calc_uplift(
 
         toc_std[it] = np.sqrt(np.mean(toc_psi[it] ** 2) / n)  # standard error of tau(q)
 
+    w = np.random.normal(0, 1, size=(n, 1000))
```

Comment:
Is 1000 the right number to use here?  I'd expect these computations to be pretty cheap so using a bigger number shouldn't be an issue if it will result in more precise esitmates, but I don't know off-hand how the precision scales with the number of replicates.

Reply:
I agree, but am also unsure of the marginal effect of increased bootstrap samples. Perhaps making it a settable parameter would be a good choice? That way users can determine the computational-precision tradeoffs for their own settings.

Reply:
Yes, I think making it configurable (and defaulting to 1000 for now) sounds reasonable.

Reply:
Great, I have made the change and pushed it. Also caught some out-of-date docstrings in the main class, which have now been updated.